mirrord/config/src/lib.rs

```diff
@@ -143,7 +143,11 @@ const PAUSE_WITHOUT_STEAL_WARNING: &str =
 ///         "ignore_localhost": false,
 ///         "unix_streams": "bear.+"
 ///       },
-///       "dns": false
+///       "dns": false,
+///       "copy_target": {
+///         "enabled": true,
```

Comment:
I think there's no `enabled` field in the config schema, right?
```suggestion
```

Reply:
Fixed :+1: